pages.es/common/git-rebase.md

```diff
@@ -4,9 +4,9 @@
 > Se utiliza comúnmente para "mover" una rama entera a otra base, ya que crea copias de los commits en una nueva ubicación.
 > Más información: <https://git-scm.com/docs/git-rebase>.
 
-- Rebasa la rama actual en lo más alto de la rama master:
+- Rebasa la rama actual en lo más alto de una rama:
```

Comment:
I'm from Argentina but I like to read docs in English :grimacing:.  `Rebasa` is the worst Spanish translation for `rebase` lol. The [official git](https://git-scm.com/book/es/v2/Ramificaciones-en-Git-Reorganizar-el-Trabajo-Realizado) use `reorganización`.
```suggestion
- Reorganiza la rama actual en lo más alto de otra rama:
```